spec/features/availability_spec.rb

```diff
@@ -65,13 +65,13 @@
   describe 'User opens a record with many holdings in a library' do
     it 'toggles 4 or more holdings with View More/View Less button', js: true do
       visit '/catalog/1793712'
-      within 'div[data-library="UP-MICRO"]' do
+      within 'div[data-library="UP-PAT"]' do
```

Comment:
Curious. Did we get rid of our micro collection then?

Reply:
not sure if we got rid of that collection but this record does not have UP-MICRO anymore